fv3core/stencils/c_sw.py

```diff
@@ -593,7 +622,7 @@ def __call__(
             self.ptc,
         )
         self._D2A2CGrid_Vectors(uc, vc, u, v, ua, va, ut, vt)
-        if self.namelist.nord > 0:
+        if self._divergence_corner is not None:
```

Comment:
It could be, but `nord > 0` doesn't give me any information about why this would not be applied. This should disappear when we can combine stencils, I think the reason it exists is that for greater nord we have to compute into the halos which involves reading from corners, but for nord = 0 we do not.

I think the "clear" way to do this with split stencils would be to determine how deep the corner needs to be computed, and calling the stencil on exactly that domain, where we sometimes call it with a domain size of 0 which results in no computation happening. But when these are fused that should happen automatically.